core/services/workflows/syncer/handler.go

```diff
@@ -532,17 +532,25 @@ func (h *eventHandler) engineFactoryFn(ctx context.Context, id string, owner str
 		return nil, fmt.Errorf("failed to get workflow sdk spec: %w", err)
 	}
 
+	// Workflow names must not exceed 10 bytes for workflow engine use.
+	// A name is used internally that is first hashed to avoid collisions,
+	// hex encoded to ensure UTF8 encoding, then truncated to 10 bytes.
```

Comment:
```suggestion
	// truncated to 10 bytes, then hex encoded to ensure UTF8 encoding.
```

Reply:
The order changed in https://github.com/smartcontractkit/chainlink-common/pull/996